tests/training/test_sae_initialization.py

```diff
@@ -179,6 +179,10 @@ def test_SparseAutoencoder_initialization_heuristic_init():
 
     decoder_norms = sae.W_dec.norm(dim=1)
 
+    # Check for both positive and negative values in the decoder weights
+    assert torch.any(sae.W_dec > 0)
+    assert torch.any(sae.W_dec < 0)
```

Comment:
```suggestion
    # Check for both positive and negative values in the decoder weights
    assert torch.any(sae.W_dec > 0)
    assert torch.any(sae.W_dec < 0)

    decoder_norms = sae.W_dec.norm(dim=1)
```

I find it more readable to have the `assert` statements with `sae.W_dec` closer to the declaration of `sae` and the declaration of `decoder_norms` closer to the `assert` statements for them.